### ISSUE TITLE
Lottie Animation Icon Resizing as per canvas size

### ISSUE THREAD

**Issue by MaanilVerma** [airbnb/lottie-web#2795]
**Tell us about your environment**
I am working in ReactJS and trying to render Lottie Animation on Canvas.

Actually, my animation is resizing according to the canvas size !! I want to maintain Canvas Size different and animation size different. i.e their sizes should not change on the basis of other one.

Here is CodeSandbox Link:-
https://6yz23p.csb.app/

**Comment by bodymovin** [comment]
can you try passing `preserveAspectRatio:"ignore"`?

**Comment by MaanilVerma** [comment]
No, didn't worked !! Actually, issue is I want no divs .. I want to render this animation on a canvas only !!
